setup/architecture.md

```diff
@@ -21,6 +21,17 @@ Coder deploys an NGINX Kubernetes ingress controller to allocate and route
 requests to the appropriate service. You can disable this controller in the helm
 chart if you use your ingress or gateway.
 
+## Ports
+
+Below is a table of the ports associated with Coder resources in the Kubernetes
```

Comment:
```suggestion
Below is a table of the listening ports associated with Coder resources in the
```

It might be worth indicating that these are inbound (listening) ports, and I don't think it's necessary to say "Kubernetes" here, we can just use "cluster"

Please also indicate in some way that these are listing our current ports and the list is subject to change

> IMO, it seems fair to document, provided that we indicate that these represent the current behavior and things may change over time. We would also need to consider how things change if we're using Istio or another service mesh

Reply:
Good catch - I will push a commit with that clarification. Thanks.